### ISSUE TITLE
[categorytemplates] Parameter must be an array or an object that implements Countable in serendipity_event_categorytemplates.php on line 792

### ISSUE THREAD

**Issue by Zugschlus** [s9y/Serendipity#725]
Hi,

I see the following message in error.log:

[Mon May 04 11:44:35.428057 2020] [proxy_fcgi:error] [pid 15939:tid 139859561801472] [client 2a01:238:42bc:a102::1c:43200] AH01071: Got error 'PHP message: PHP Warning:  count(): Parameter must be an array or an object that implements Countable in /home/mh/blog.zugschlus.de/Serendipity/plugins/serendipity_event_categorytemplates/serendipity_event_categorytemplates.php on line 792', referer: https://newblog.zugschlus.de/index.php?/archives/1043-Warum-Zugschlus.html

I have locally solved this by deinsalling the unused categorytemplates plugin from my installation, but maybe this is easily reproduced and fixed in dev.

Grettings
Marc

**Comment by th-h** [comment]
Yep, thanks. We'll add that to our growing list of PHP 7.x incompatibilities in older plugins. :-)
